srml/democracy/src/vote_threshold.rs

```diff
@@ -69,14 +73,38 @@ fn compare_rationals<T: Zero + Mul<T, Output = T> + Div<T, Output = T> + Rem<T,
 	}
 }
 
-impl<Balance: IntegerSquareRoot + Zero + Ord + Add<Balance, Output = Balance> + Mul<Balance, Output = Balance> + Div<Balance, Output = Balance> + Rem<Balance, Output = Balance> + Copy> Approved<Balance> for VoteThreshold {
-	/// Given `approve` votes for and `against` votes against from a total electorate size of
-	/// `electorate` of whom `voters` voted (`electorate - voters` are abstainers) then returns true if the
-	/// overall outcome is in favor of approval.
+impl<Balance> Approved<Balance> for VoteThreshold
+	where Balance: IntegerSquareRoot
```

Comment:
suggest this be included in a separate PR since it changes the implementation (instead of mixed into a documentation-specific PR)